apps/examples/alc5658char_test/Makefile

```diff
@@ -60,7 +60,7 @@ include $(APPDIR)/Make.defs
 # built-in application info
 
 APPNAME = alctest
-FUNCNAME = alc5658char_test_main
+REALNAME = alc5658char_test
```

Comment:
What is realname? Why do you use this definition?

Reply:
reverted.